src/scripts/main.js

```diff
@@ -1,5 +1,28 @@
 'use strict';
+const wall = document.querySelector('.wall');
 
-document.addEventListener('click', (e) => {
-  // write code here
+wall.addEventListener('click', (e) => {
+  const spider = document.querySelector('.spider');
+  const spiderRect = spider.getBoundingClientRect();
```

Comment:
You are querying the spider element inside the click event. This means every time a click event occurs, the browser will search for the spider element in the DOM. This is not efficient. It would be better to query the spider element outside of the event listener, store it in a variable, and then use that variable inside the event listener.